ui/app/upload_image/controller.js

```diff
@@ -15,11 +15,17 @@ const uploadFile = multer({
 const handleUploadImageRoute = [
     uploadFile.single('file'),
     (request, response, next) => {
-        const file = request.file
         const token = request.session.jwt
-        handlePostingImageToAPI(file, token).then((result) => {
-            if (result['sloth_check'].contains_sloth) {
-                response.status(200).render('index', {page: 'Slothbucket', contains_sloth: 'Contains a sloth!'})
+        const file = request.file
+        handleUploadingImage(file, token).then((image) => {
+            const base64 = image.base64
+            if (image['sloth_check'].contains_sloth) {
+                // TODO: upon setting up a relational table, we can next or redirect to a page that will load all
+                // of the user's pictures, including the one just uploaded
+                response.status(200).render('index', {
+                    page: 'Slothbucket',
+                    contains_sloth: 'Contains a sloth!',
```

Comment:
You might want to call this variable something else. The naming scheme implies it's gonna be a variable. With a dynamically typed language you want to be careful of misnomers.

Also, since this response logic is very close to the one in the `else` block, you could combine them and use ternaries to set the `contains_sloth` and `images` field. It's fine if the `images` field is undefined